### PR TITLE
Add env variable support

### PR DESCRIPTION
**Pull request by rodxavier**
Solves #4 via `from_env()` class method

### CODE REVIEW

File: src/client.py [hunk 1]
```diff
@@ -59,6 +60,19 @@ def __init__(self, url, access_key, secret_key):
         self.session = self.__requests_retry_session()
         self.session_not_found_ok = self.__requests_retry_not_found_ok_session()
 
+    @classmethod
+    def from_env(cls):
```
Comment (rebstar6) [inline src/client.py:64]: pydocstyle issue here:
```
src/client.py:64 in public method `from_env`:
        D102: Missing docstring in public method
```

Comment (rodxavier) [reply src/client.py:64]: added the missing docstring @rebstar6

**Comment by rebstar6** [comment]
Thanks @rodxavier, dont worry about the flake failures, other PRs are aiming to address those.

File: src/client.py [hunk 2]
```diff
@@ -59,6 +60,24 @@ def __init__(self, url, access_key, secret_key):
...
+        """
+        Create client from environtment variables.
+
+        :return: a client instance
+        """
+        url = cls.__get_from_env('VINYLDNS_API_URL')
```
Comment (rebstar6) [inline src/client.py]: could you check for all 3 at once rather than one at a time? would be helpful to know up front if all 3 (or 2 of 3) are missing

Comment (rodxavier) [reply src/client.py]: changed how env variables are checked @rebstar6

**Comment by rebstar6** [comment]
thanks @rodxavier, looks good! we will be making more issues in this repo in the coming days - if youre interested in picking up more, give it a look or reach out in gitter (https://gitter.im/vinyldns/Lobby)!

**Review by rebstar6 — APPROVED** [review]

**Pull request merged**